diff --git a/ogre2/src/Ogre2RenderEngine.cc b/ogre2/src/Ogre2RenderEngine.cc
index 60b89bdc..6c511a92 100644
--- a/ogre2/src/Ogre2RenderEngine.cc
+++ b/ogre2/src/Ogre2RenderEngine.cc
@@ -764,7 +764,9 @@ void Ogre2RenderEngine::CreateRenderSystem()
   Ogre::RenderSystem *renderSys;
   const Ogre::RenderSystemList *rsList;
 
+  gzerr << "11111" << std::endl;
   rsList = &(this->ogreRoot->getAvailableRenderers());
+  gzerr << "rlist size: " << rslist.size() << std::endl;
   std::string targetRenderSysName("OpenGL 3+ Rendering Subsystem");
   if (this->dataPtr->graphicsAPI == GraphicsAPI::VULKAN)
   {
@@ -775,6 +777,8 @@ void Ogre2RenderEngine::CreateRenderSystem()
     targetRenderSysName = "Metal Rendering Subsystem";
   }
 
+  gzerr << targetRenderSysName << std::endl;
+
   int c = 0;
 
   renderSys = nullptr;
@@ -797,7 +801,7 @@ void Ogre2RenderEngine::CreateRenderSystem()
         renderSys && renderSys->getName().compare(targetRenderSysName) != 0;
     } while (bContinue);
   }
-
+  gzerr << "222222" << std::endl;
   if (renderSys == nullptr)
   {
     gzerr << "unable to find " << targetRenderSysName << ". OGRE is probably "
@@ -809,7 +813,9 @@ void Ogre2RenderEngine::CreateRenderSystem()
   {
 
     // We operate in windowed mode
+    gzerr << "333333333" << std::endl;
     renderSys->setConfigOption("Full Screen", "No");
+    gzerr << "44444444" << std::endl;
 
     if (this->dataPtr->graphicsAPI == GraphicsAPI::OPENGL)
     {
@@ -819,7 +825,9 @@ void Ogre2RenderEngine::CreateRenderSystem()
       ///   PBuffer limits the size of the renderable area of the RTT to the
       ///           size of the first window created.
       ///   FBO is the only good option
+      gzerr << "555555555555" << std::endl;
       renderSys->setConfigOption("RTT Preferred Mode", "FBO");
+      gzerr << "66666666666" << std::endl;
     }
   }
   else
@@ -837,12 +845,16 @@ void Ogre2RenderEngine::CreateRenderSystem()
   }
 
   // get all supported fsaa values
+  gzerr << "7777777777" << std::endl;
   Ogre::ConfigOptionMap configMap = renderSys->getConfigOptions();
+  gzerr << "8888888888888" << std::endl;
   auto fsaaOoption = configMap.find("FSAA");
-
+  gzerr << "999999999999" << std::endl;
   if (fsaaOoption != configMap.end())
   {
+    gzerr << "AAAAAAAAAAA" << std::endl;
     auto values = (*fsaaOoption).second.possibleValues;
+    gzerr << "BBBBBBBBBBB" << std::endl;
     for (auto const &str : values)
     {
       int value = 0;
@@ -857,6 +869,7 @@ void Ogre2RenderEngine::CreateRenderSystem()
       this->dataPtr->fsaaLevels.push_back(value);
     }
   }
+  gzerr << "CCCCCCCCCCCC" << std::endl;
   std::sort(this->dataPtr->fsaaLevels.begin(), this->dataPtr->fsaaLevels.end());
 
   // check if target fsaa is supported
@@ -869,7 +882,9 @@ void Ogre2RenderEngine::CreateRenderSystem()
 
   renderSys->setConfigOption("FSAA", std::to_string(fsaa));
 
+  gzerr << "DDDDDDDDD" << std::endl;
   this->ogreRoot->setRenderSystem(renderSys);
+  gzerr << "EEEEEEEEE" << std::endl;
 }
 
 void Ogre2RenderEngine::RegisterHlms()
