diff --git a/Components/SceneFormat/src/OgreSceneFormatExporter.cpp b/Components/SceneFormat/src/OgreSceneFormatExporter.cpp
index f990121cd2..27d1f7a90f 100644
--- a/Components/SceneFormat/src/OgreSceneFormatExporter.cpp
+++ b/Components/SceneFormat/src/OgreSceneFormatExporter.cpp
@@ -64,11 +64,11 @@ THE SOFTWARE.
 #include <fstream>
 
 #if OGRE_COMPILER == OGRE_COMPILER_MSVC && OGRE_COMP_VER < 1800
-    inline float isfinite( float x )
+    inline float std::isfinite( float x )
     {
         return _finite( x ) == 0;
     }
-    inline float isinf( float x )
+    inline float std::isinf( float x )
     {
         return  x == std::numeric_limits<float>::infinity() ||
                 x == -std::numeric_limits<float>::infinity();
@@ -159,11 +159,11 @@ namespace Ogre
             strValue.a( encodeFloatBin( value ) );
         else
         {
-            if( isfinite( value ) )
+            if( std::isfinite( value ) )
                 strValue.a( LwString::Float( value, 9 ) );
             else
             {
-                if( isinf( value ) )
+                if( std::isinf( value ) )
                     strValue.a( value > 0 ? "\"inf\"" : "\"-inf\"" );
                 else
                     strValue.a( "\"nan\"" );
@@ -183,11 +183,11 @@ namespace Ogre
             strValue.a( encodeDoubleBin( value ) );
         else
         {
-            if( isfinite( value ) )
+            if( std::isfinite( value ) )
                 strValue.a( LwString::Float( value, 18 ) );
             else
             {
-                if( isinf( value ) )
+                if( std::isinf( value ) )
                     strValue.a( value > 0 ? "\"inf\"" : "\"-inf\"" );
                 else
                     strValue.a( "\"nan\"" );
